### DISCUSSION TITLE
to_json and from_json skip missing elements.

### DISCUSSION THREAD

**Discussion by Gringoniceguy** [nlohmann/json#2465] [Q&A] [answered]
Say we have the following code
```
struct person
{
    std::string name = "";
    int number = 0;
    NLOHMANN_DEFINE_TYPE_INTRUSIVE(person, name, number)
}
```
Now this is going to work fine now lets say we write this to a file.
We now add a another variable `std::string address = "cool street";`.
We still want the old data to work we just want address to be it's default, in this case "cool street".
Is there such a option to just skip element if it does not exist?

**Comment by nlohmann** [comment]
For this, you need to define your own `to_json` function to do so.,

**Comment by Gringoniceguy** [comment, marked as answer]
I found a better fix by just changing #define NLOHMANN_JSON_FROM(v1)
to just be wraped in a try catch.
